src/common/log.js

```diff
@@ -11,30 +12,39 @@ const logger = bunyan.createLogger({
   process: process.type
 })
 
+function logToStdout() {
+  logger.addStream({
+    stream: process.stdout,
+    level: 'debug'
+  })
+}
 
-function init(dir) {
+function logToFolder(dir) {
+  if (dir) {
+    let logDir = join(dir, 'log')
 
+    // ensure logging dir exists
+    if (!existsSync(logDir)) mkdirSync(logDir)
+
+    logger.addStream({
+      type: 'rotating-file',
+      path: join(logDir, `${process.type}.log`),
```

Comment:
Just a heads up that we'll have to make this configurable (we will have multiple 'renderer' processes using the logger).